rfcs/websockets_http2.md

```diff
@@ -0,0 +1,24 @@
+# RFC #59: Add WebSockets over HTTP/2 support to wptserve
+
+## Summary
+
+Enable wptserve to support WebSockets over an existing http2 connection.
+
+## Details
+
+Allow tests to open a WebSocket over an existing http2 connection.
+
+wptserver needs to implement the [Bootstrapping WebSocket with HTTP/2 protocol](https://tools.ietf.org/html/rfc8441).
+
+It means that it needs to be modified to:
+- Set ENABLE_CONNECT_PROTOCOL settings to 1
+- Detect Extended CONNECT Method and implement the WebSocket opening handshake.
+- Wrap H2Response and H2Request into a Request-like object that pywebsocket can operate on.
```

Comment:
Should point out which `Request` this is (the standard `http.client` library? our own in `wptserve`? or something else?).